Include/internal/pycore_object.h

```diff
@@ -38,11 +38,12 @@ static inline void _PyObject_GC_TRACK_impl(const char *filename, int lineno,
                           "object is in generation which is garbage collected",
                           filename, lineno, "_PyObject_GC_TRACK");
 
-    PyGC_Head *last = (PyGC_Head*)(_PyRuntime.gc.generation0->_gc_prev);
+    PyInterpreterState *interp = _PyInterpreterState_GET_UNSAFE();
```

Comment:
Would it be possible to use a "struct _gc_runtime_state *state" variable instead?

```
struct _gc_runtime_state *state = &_PyInterpreterState_GET_UNSAFE()->gc;
```